frontend/app/routes-flow/apply-flow.ts

```diff
@@ -68,6 +68,18 @@ async function loadState({ params, request, fallbackRedirectUrl = '/apply' }: Lo
   }
 
   const state = session.get(sessionName) as ApplyState;
+
+  if (Date.now() - session.get('timeUpdated') > 15 * 60 * 1000) {
+    session.unset(sessionName);
+    session.unset('timeUpdated');
+    throw redirectWithLocale(request, fallbackRedirectUrl, {
+      status: 302,
```

Comment:
`status: 302` is unnecessary here since the underlying `redirectWithLocale()` function calls remix's `redirect()` function.